features/step_definitions/basic_steps.rb

```diff
@@ -10,3 +10,7 @@
   click_link_or_button link
 end
 
+Given("I visit {string} page") do |page|
+  visit cart_path(:id)
```

Comment:
This step definition is not making use of the variable. There are step definitions like this one already. Why not reuse them using 'case'?

Reply:
Also, where does :id come from?

Reply:
This :id should be calling on the carts id will take a better look